src/pages/Deeplink.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div class="px-20 py-6">
+
+    <div class="flex items-center justify-between">
+      <div class="flex items-center justify-center">
+        <breadcrumbs :name="$t('sidebar.deeplink')" />
+      </div>
+    </div>
+
+    <div class="mr-5 py-5">
+      <label
```

Comment:
Graet stuff, just a few more changes, for the inputs use those similar those in `FeatureRequest.vue` and `Home.vue`, no need to change the button though, the one in `FeatureRequest.vue` is the disabled version:

![feature-request](https://i.imgur.com/Ovm0EMu.png)

Your form container should be:

```html
<div class="px-4 py-3 mb-8 bg-white rounded-lg shadow-md dark:bg-gray-800">
...
</div>
```